### DISCUSSION TITLE
Working implementations or examples of @accounts/oauth-twitter

### DISCUSSION THREAD

**Discussion by gezquinndesign** [accounts-js/accounts#1127] [Q&A] [answered]
I've managed to get a login with Google button working on my site using the example [here](https://github.com/accounts-js/accounts/pull/961).  My next task is to implement a Log in with Twitter button but I'm struggling to find any good examples of how to set it up (including the frontend requirements) using the `@accounts/oauth-twitter` adapter.

Does anyone have any good examples or could anyone give me any pointers please?

**Comment by gezquinndesign** [comment, marked as answer]
Using the same approach as the Google button, I managed to get the Twitter one working too, though I had to add an additional getRequestToken method to my class. If anyone is reading this and wants to know specifics, get in touch.

**Reply by sanguinary-bus** [reply]
can you share your approach?
oauth example is vague
